src/utils/layerUtils.js

```diff
@@ -76,7 +78,7 @@ export function getWmtsXyzUrl(wmtsLayerConfig, projection, options = {}) {
         const layerId = wmtsLayerConfig.isExternal
             ? wmtsLayerConfig.id
             : wmtsLayerConfig.technicalName
-        return `${wmtsLayerConfig.baseUrl}1.0.0/${layerId}/default/${timestamp}/${projection.epsgNumber}/{z}/{x}/{y}.${wmtsLayerConfig.format}`
+        return `${baseUrlOverride ?? wmtsLayerConfig.baseUrl}1.0.0/${layerId}/default/${timestamp}/${projection.epsgNumber}/{z}/{x}/{y}.${wmtsLayerConfig.format}`
```

Comment:
Why using baseUrlOverride here ? it should be `getWmtsBaseUrl`, a reason to not export the baseUrlOverride to avoid such bug.

Reply:
I might have forgotten to change this one, should be good now